### DISCUSSION TITLE
Cost estimation?

### DISCUSSION THREAD

**Discussion by mikaelkundert** [mysticaltech/terraform-hcloud-kube-hetzner#479] [Q&A] [answered]
Hi! This looks interesting project.

What is the minimum set of resources needed to run minimum cluster (no apps running, just the cluster)?

Just to calculate a estimation of monthly cost.

**Comment by captnCC** [comment, marked as answer]
The minimum cluster would consist of a single `CPX11` node clocking in at 4,58 Euro/month.
If the cluster should have all HA features a set of 3 control planes, 3 agents and load balancer should be used. This would mean 34,09 per month

**Reply by schlichtanders** [reply]
@captnCC I am also looking for creating a minimal cluster and am surprised to find that the Hetzner Load Balancer costs too.
It is not listed in your answer, have you found a way to use kubernetes ingress directly without an extra hetzner load balancer?
